content/departments/product-engineering/process/cross-org-team-collab.md

```diff
@@ -0,0 +1,19 @@
+# Cross-org/team collaboration
+
+Sometimes, ideas (and even code) for something your team owns comes from other teams, orgs, or even departments at Sourcegraph. That's a good thing because everyone can contribute to the innovation and success of the product we all use, and the more we do of that sort of thing, the more of a flywheel we're generating for improvements and is one of the real advantages of being a software development company that makes products for software developers.
+
+That said, if the collaboration part of this is not handled well, you can end up in situations where we create duplicative product features, or waste time building things that eventually will need to be reworked, or even shut down.
+
+To avoid this, as an open core company we can look to open source to see how this kind of situation is handled:
+
+1. There should always be a clear Product & Engineering team `maintainer` who makes the final decision of whether to accept a suggestion/code from another team. You can refer to the [ownership chart](../engineering/process/engineering_ownership.md) to determine who the maintainer is, and they are the first person you should go to to discuss your idea.
```

Comment:
```suggestion
1. There should always be a clear Product & Engineering team `maintainer` who makes the final decision of whether to accept a suggestion/code from another team. You can refer to the [ownership chart](../engineering/process/engineering_ownership.md) to determine who the maintainer is, and they are the first person you should go to to discuss your idea. In cases where the question isn't directly dependent on or modifying existing code, you should refer to the "Vision" sections of [team strategy pages](https://handbook.sourcegraph.com/strategy-goals/strategy/#team-strategy-pages) to determine if there is already a `maintainer` of features related to the job-to-be-done you are pursuing. 
```

(I think in general the open source model is a great lens to use for this PR!) 

Making this suggestion because I think this problem is most likely to arise when there is not clear "code ownership" and that ownership chart [live version](https://handbook.sourcegraph.com/departments/product-engineering/engineering/process/engineering_ownership/) is mostly about technical things, not user flows/jobs to be done. 

With compute<>insights, and soon/in the future with insights PR data tracking<>batch changes potential overlap, this ownership doesn't appear on the chart because it's about "jobs to be done" of the feature (for the former, "treating your codebase as a database to answer high level questions"; for the latter "PR tracking use cases"). 

But I think we need some way to document/determine "product area ownership." 

This type of jobs-to-be-done ownership generally appears on our strategy pages most succinctly in the vision section. For example: https://handbook.sourcegraph.com/strategy-goals/strategy/code-graph/batch-changes/#3-year-vision says the 3-year vision is: 
> Developers can easily run code to create changesets across many repositories and codehosts, and track them to completion.

and now as a PM I can easily understand "oh, if I am working on something to create changesets across many repos, or that tracks them, I should check with the batch changes team as the maintainer."

Reply:
Makes sense, committing the suggestion.